docs/development/doc_guide.rst

```diff
@@ -1,76 +1,633 @@
+The following code block contains a few common examples of Markdown
+formatting.
+
+.. code-block:: markdown
+
+   # Header 1
+
+   ## Header 2
+
+   Here is a link to [PlasmaPy's documentation](https://docs.plasmapy.org).
+
+   We can make text **bold** or *italic*.
+
+   We can write in-line code like `x = 1` or create a Python code block:
+
+   ```Python
+   y = 2
+   z = 3
+   ```
+
+Sphinx
+------
+
+`sphinx` is the software used to generate `PlasmaPy's online documentation`_
+from reST files and Python docstrings.
+
+Configuration
+~~~~~~~~~~~~~
+
+The `docs/conf.py`_ file contains the configuration information needed
+to customize `sphinx` behavior.
+The documentation for `sphinx` lists the `configuration options
+<https://www.sphinx-doc.org/en/master/usage/configuration.html>`_
+that can be set.
+
+Sphinx extensions
+~~~~~~~~~~~~~~~~~
+
+PlasmaPy documentation is built with the following `sphinx` extensions:
+
+* `sphinx.ext.autodoc
+  <https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html>`_
+  for including documentation from docstrings
+* `sphinx.ext.intersphinx
+  <https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html>`_
+  for linking to other projects' documentation
+* `sphinx.ext.graphviz
+  <https://www.sphinx-doc.org/en/master/usage/extensions/graphviz.html>`_
+  to allow `Graphviz <https://graphviz.org/>`_ graphs to be included
+* `sphinx.ext.mathjax
+  <https://www.sphinx-doc.org/en/master/usage/extensions/math.html#module-sphinx.ext.mathjax>`_
+  for math rendering with `MathJax <https://www.mathjax.org/>`_
+* `sphinx.ext.napoleon
+  <https://www.sphinx-doc.org/en/master/usage/extensions/napoleon.html>`_
+  for allowing NumPy style docstrings
+* `sphinx.ext.todo
+  <https://www.sphinx-doc.org/en/master/usage/extensions/todo.html>`_ to support
+  ``todo`` directives
+* `nbsphinx <https://nbsphinx.readthedocs.io>`_ for including
+  `Jupyter`_ notebooks
+* `sphinx_copybutton <https://sphinx-copybutton.readthedocs.io>`_ to add
+  a "copy" button for code blocks
+* `sphinx_gallery.load_style
+  <https://sphinx-gallery.github.io/stable/advanced.html?highlight=load_style#using-only-sphinx-gallery-styles>`_
+  for using sphinx-gallery styles
+* IPython.sphinxext.ipython_console_highlighting
+* `sphinx_changelog <https://sphinx-changelog.readthedocs.io>`_
+  for rendering `towncrier`_ changelogs
+* `plasmapy_sphinx` for customizations created for use in PlasmaPy.
+  Note that `plasmapy_sphinx` is expected to be broken out into its own
+  package at a future date.
+
+These extensions are specified in ``extensions`` in `docs/conf.py`_.
+
+References to other packages
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Intersphinx_ allows the automatic generation of links to the
+documentation of objects in other projects. The mappings are defined in
+the ``intersphinx_mapping`` dictionary in `docs/conf.py`_, and include
+`python`, `numpy`, `scipy`, `astropy`, `pandas`, `sphinx`, and
+`sphinx_automodapi`.
```

Comment:
```suggestion
Intersphinx_ allows the automatic generation of links to the
documentation of objects in other projects. The mappings are defined in
the :confval:`intersphinx_mapping` dictionary in `docs/conf.py`_, and include
`python`, `numpy`, `scipy`, `astropy`, `pandas`, `sphinx`, and
`sphinx_automodapi`.
```

What I said in https://github.com/PlasmaPy/PlasmaPy/pull/1156/files?file-filters%5B%5D=.rst#r654641786.  Should propagate this to all references of `intersphinx_mappings`.